content/en/docs/ops/integrations/spire/index.md

```diff
@@ -0,0 +1,408 @@
+for more information on configuring SPIRE to integrate with Istio deployments.
+
+#### SPIRE CA Integration Prerequisites
+
+To integrate your SPIRE deployment with Istio, configure SPIRE:
+
+1. Access [SPIRE Agent reference](https://spiffe.io/docs/latest/deploying/spire_agent/#agent-configuration-file) and
+   configure the SPIRE Agent socket path to match the Envoy SDS defined socket path.
+
+{{< text plain >}}
+socket_path = "/run/secrets/workload-spiffe-uds/socket"
+{{< /text >}}
+
+2. Share the SPIRE Agent socket with the pods within the node by deploying the
+   [SPIFFE CSI Driver](https://github.com/spiffe/spiffe-csi).
+
+3. See [Install Istio](#install-istio) to configure Istio to integrate with the SPIFFE CSI Driver.
+
+{{< warning >}}
+Note that you must deploy SPIRE before installing Istio into your environment so that Istio can detect it as a CA. 
+{{< /warning >}}
+
+## Install Istio
+
+1. [Download the latest Istio release](https://istio.io/latest/docs/setup/getting-started/#download). 
+
+2. After [deploying SPIRE](#install-spire) into your environment, and verifying that all deployments are in Ready state,
+install Istio with custom patches for istio-ingressgateways/egressgateways as well as the istio-proxy.
+
+{{< text bash >}}
+$ istioctl install --skip-confirmation -f - <<EOF
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+metadata:
+  namespace: istio-system
+spec:
+  profile: default
+  meshConfig:
+    trustDomain: example.org
+  values:
+    global:
+    # This is used to customize the sidecar template
+    sidecarInjectorWebhook:
+      templates:
+        spire: |
+          spec:
+            containers:
+            - name: istio-proxy
+              volumeMounts:
+              - name: workload-socket
+                mountPath: /run/secrets/workload-spiffe-uds
+                readOnly: true
+             volumes:
+               - name: workload-socket
+                 csi:
+                   driver: "csi.spiffe.io"
+  components:
+    ingressGateways:
+      - name: istio-ingressgateway
+        enabled: true
+        label:
+          istio: ingressgateway
+        k8s:
+          overlays:
+            - apiVersion: apps/v1
+              kind: Deployment
+              name: istio-ingressgateway
+              patches:
+                - path: spec.template.spec.volumes.[name:workload-socket]
+                  value:
+                    name: workload-socket
+                    csi:
+                      driver: "csi.spiffe.io"
+                - path: spec.template.spec.containers.[name:istio-proxy].volumeMounts.[name:workload-socket]
+                  value:
+                    name: workload-socket
+                    mountPath: "/run/secrets/workload-spiffe-uds"
+                    readOnly: true
+EOF
+{{< /text >}}
+
+3. Use [Automatic sidecar injection](https://istio.io/latest/docs/setup/additional-setup/sidecar-injection/#automatic-sidecar-injection)

```

Comment:
I think that this paragraph talking only and specifically about automatic sidecar injection when referring to the injection of the `spire` template might confuse the reader, as there might be room for thinking that manual sidecar injection wouldn't work? IDK, might be being too picky on this one, but I think if we're not going to talk about both ways, we could just talk about it generically (not saying automatic or manual). My suggestion:

```
Use [sidecar injection](https://istio.io/latest/docs/setup/additional-setup/sidecar-injection) to inject the `istio-proxy`
container into the pods within your mesh. See [custom templates](https://istio.io/latest/docs/setup/additional-setup/sidecar-injection/#custom-templates-experimental) 
for information on how to apply the custom defined `spire` template into `istio-proxy`. This allows for the CSI driver to mount the UDS on the sidecars.
```